test/mocks/data/map_pin.dart

```diff
@@ -0,0 +1,18 @@
+import "package:google_maps_flutter/google_maps_flutter.dart";
+import "package:proxima/models/ui/map_pin.dart";
+
+class MapPinGenerator {
+  //generate a list of MapPin
+  List<MapPin> generateMapPins(int numberOfPins) {
```

Comment:
Nice work implementing the generator !
I just think that we can make this method static :+1:

Reply:
Fixed with [7e338be](https://github.com/ProximaEPFL/proxima/pull/231/commits/7e338bed5c379b7d804b3ce087536e3634ffe632)